eng/common/performance/android_scenarios.proj

```diff
@@ -1,34 +0,0 @@
-<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
```

Comment:
@billwert the changes to eng/common from https://github.com/dotnet/runtime/pull/50407 need to flow through dotnet/arcade otherwise they'll be overwritten.

Reply:
good catch! I will jam them in there right now.

Reply:
PR is up and approved: https://github.com/dotnet/arcade/pull/7178.